app/views/home/index.html.erb

```diff
@@ -0,0 +1,11 @@
+<div class="home-c">
+
+<h1 class="home-h1">Recipe App 🍴</h1>
+<p class="home-p">
+The Recipe app keeps track of all your recipes, ingredients, and inventory. It will allow you to save ingredients, keep track of what you have, create recipes, and generate a shopping list based on what you have and what you are missing from a recipe. Also, since sharing recipes is an important part of cooking the app allows you to make them public so anyone can access them.
+</p>
+
+<p class="home-p">
+The Recipe app keeps track of all your recipes, ingredients, and inventory. It will allow you to save ingredients, keep track of what you have, create recipes, and generate a shopping list based on what you have and what you are missing from a recipe. Also, since sharing recipes is an important part of cooking the app allows you to make them public so anyone can access them.
```

Comment:
## Optional
- Please adjust the font color of the navbar to the link pages' font color. This will help your design look nice and avoid giving eye pain to users. 
- 
![navbar](https://user-images.githubusercontent.com/63972670/187642412-ac1a62ec-3d94-46d1-ac02-a28d81aaccd8.JPG)

- Please work on the logging font color.

- 
![registration](https://user-images.githubusercontent.com/63972670/187642981-f8b587ba-ee44-461a-ae0f-19746998bc3d.JPG)